index.js

```diff
@@ -437,12 +437,14 @@ class HtmlWebpackPlugin {
       }
 
       // Webpack outputs an array for each chunk when using sourcemaps
-      // But we need only the entry file
+      // But we need only the initial entry file in case it is a js file
       const entry = chunkFiles[0];
-      assets.chunks[chunkName].size = chunk.size;
-      assets.chunks[chunkName].entry = entry;
-      assets.chunks[chunkName].hash = chunk.hash;
-      assets.js.push(entry);
+      if (/.css($|\?)/.test(entry)) {
```

Comment:
```js
if (!/\.css$/.test(entry))
```

Otherwise it still pushes the CSS entry to `assets.js` ?

Reply:
sorry totally right